book/src/csi-proxy.md

```diff
@@ -0,0 +1,44 @@
+# CSI Proxy
+
+## Status and Releases
+
+**Git Repository:** [https://github.com/kubernetes-csi/csi-proxy](https://github.com/kubernetes-csi/csi-proxy)
+
+**Status:** Beta starting with v0.2.0
+
+Status | Min K8s Version | Max K8s Version
+--|--|--
+v0.1.0 | 1.18 | - 
+v0.2.0+ | 1.18  | -
+
+## Description
+
+CSI Proxy is a binary that exposes a set of gRPC APIs around storage operations over named pipes in Windows. A container, such as CSI node plugins, can mount the named pipes depending on operations it wants to exercise on the host and invoke the APIs.
+
+Each named pipe will support a specific version of an API (e.g. v1alpha1, v2beta1) that targets a specific area of storage (e.g. disk, volume, file, SMB, iSCSI). For example, `\\.\pipe\csi-proxy-filesystem-v1alpha1`, `\\.\pipe\csi-proxy-disk-v1beta1`
```

Comment:
Is there a place where every pipe is actually documented and we can point to?

Reply:
not really, just a short explanation on csi-proxy readme. https://github.com/kubernetes-csi/csi-proxy

Reply:
Can we open an issue to investigate if we can have some api reference generator?

Reply:
yes, we are checking on it